packages/react/src/ActionList/Description.tsx

```diff
@@ -37,11 +42,11 @@ export const Description: React.FC<React.PropsWithChildren<ActionListDescription
 
   const {blockDescriptionId, inlineDescriptionId} = React.useContext(ItemContext)
 
-  return variant === 'block' ? (
+  return variant === 'block' || !truncate ? (
     <Box
       as="span"
       sx={merge(styles, sx as SxProp)}
-      id={blockDescriptionId}
+      id={variant === 'block' ? blockDescriptionId : inlineDescriptionId}
```

Comment:
I didn't know we have different IDs for inline and block 💭 good to know

Reply:
was surprised by this one as well hahaah